packages/grid/x-grid/src/tests/columnHeaders.DataGrid.test.tsx

```diff
@@ -36,8 +36,8 @@ describe('<XGrid /> - Column Headers', () => {
     ],
   };
 
-  describe('GridColumnHeaderMenu', () => {
-    it('should close menu when resizing a column', async () => {
+  describe.only('GridColumnHeaderMenu', () => {
```

Comment:
don't forget to remove the `.only`

Reply:
I just removed it
I did not know this feature, it's great but I'm forgetting to remove it all the time :grimacing:

Reply:
eslint will fail the build if it's still around